VultisigApp/VultisigApp/Model/Coin.swift

```diff
@@ -127,12 +127,14 @@ class Coin: ObservableObject, Codable, Hashable {
             return "2000000000"
         case .solana:
             return "7000"
-        case .ethereum,.avalanche,.base,.blast,.arbitrum,.polygon,.optimism,.bscChain,.cronosChain, .zksync:
+        case .ethereum,.avalanche,.base,.blast,.arbitrum,.polygon,.optimism,.bscChain,.cronosChain:
             if self.isNativeToken {
-                return "23000"
+                return "40000"
```

Comment:
this, isn't it going to cause we left more in the address?

Reply:
In my tests, it left no dust for most of the EVMs. L1 and L2 behave differently because of their optimization mechanisms. Maybe I can try 23k + 10k for L1 and 40k for L2s. 

**Test Results**
Arb.ETH: OK 100%, no dust
Base.ETH: OK 100%, no dust
ETH.ETH: OK 100%, some dust
ETH.RENDER: OK 100%, no dust
ETH.UNI: OK 100%, no dust
ZK.ETH: OK, no dust but uses 200k gas (It is calculated differently)